docs/advanced/distributed-caching.md

```diff
@@ -0,0 +1,45 @@
+# Distributed Caching
+
+Status: **General Availability**
+
+When a function returns a value, Differential will store it in the control-plane state. It's then available to be used by other calls, without having to call the function again. This is useful for caching expensive operations, and speeding up your application.
```

Comment:
Can we swap the two sentences?

Differential is useful for caching expensive operations, and speeding up your application by storing returned function values in the control-plane. It is then available to be used by other calls without having to call that specific function again


(Value + How) : also it flows better with the next part. cause currently, it's how, then value, then the next paragraph is how again